webcurator-ui/src/views/target/target-tabs/TargetTabPanelGeneral/TargetTabPanelGeneralSeeds.vue

```diff
@@ -84,50 +90,79 @@ watch(() => props.editing, async(newEditing) => {
       </div>
     </div>
 
-    <DataTable class="w-full" :rowHover="true" :value="targetSeeds.targetSeeds">
-      <Column field="seed" header="Seed">
-        <template #body="{ data }">
-          <div class="flex align-items-center justify-content-between">
+    <table class="w-full target-seed-parent-table">
+      <thead>
+        <tr style="border-bottom: 1px solid #E4E4E4;">
+          <th style="width: 20%; text-align: left; padding: .5rem;">Seed</th>
+          <th style="width: 10%; text-align: left; padding: .5rem;">Primary</th>
+          <th style="width: 15%; text-align: left; padding: .5rem;">Harvest Auth</th>
+          <th style="width: 15%; text-align: left; padding: .5rem;">Auth Agent</th>
+          <th style="width: 17.5%; text-align: left; padding: .5rem;">Start Date</th>
+          <th style="width: 17.5%; text-align: left; padding: .5rem;">End Date</th>
+          <th style="width: 10%; text-align: left; padding: .5rem;">Actions</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="(data, index) in targetSeeds.targetSeeds" :key="index">
+          <td style="width: 20%; padding: .5rem;">
             <span v-if="editingSeed != data.id">{{ data.seed }}</span>
             <InputText v-else v-model="data.seed" />
-          </div>
-        </template>
-      </Column>
-      <Column field="authorisations" header="Harvset Auth">
-        <template #body="{ data }">
-          <div class="flex align-items-center">
-            <div>
-              <div v-for="authorisation in data.authorisations" :key="authorisation.id">
-                {{ authorisation }}
-              </div>
-            </div>
-            <div class="flex-shrink-0">
-              <Button v-if="editing && editingSeed == data.id" label="+/-" text @click="showAddHarvestAuth(data)" />
+          </td>
+          <td style="width: 10%; padding: .5rem;">
+            <span v-if="editingSeed != data.id">{{ data.primary ? 'Yes' : 'No' }}</span>
+            <Checkbox v-else v-model="data.primary" :binary="true" />
+          </td>
+          <td colspan="5" style="padding: .5rem;">
+            <table class="w-full target-seed-child-table">
+              <tbody>
+                <tr v-for="authorisation in data.authorisations" :key="authorisation.id">
+                  <td style="width: 15%; padding: .5rem;">{{ authorisation.name }}</td>
+                  <td style="width: 15%; padding: .5rem;">{{ authorisation.agent }}</td>
+                  <td style="width: 17.5%; padding: .5rem;">{{ formatDate(authorisation.startDate) }}</td>
+                  <td style="width: 17.5%; padding: .5rem;">{{ formatDate(authorisation.endDate) }}</td>
```

Comment:
If the value of Date is null, "01/01/1970" will be shown. Empty string is required for the null/undefined values.